packages/subgraph/subgraph.yaml

```diff
@@ -7,11 +7,11 @@ features:
 dataSources:
   - kind: ethereum/contract
     name: Factory
-    network: gnosis
+    network: mumbai
```

Comment:
Why did gnosis get replaced here?

Reply:
That's a side effect of how the TheGraph's library handles the `--network` parameter passed when deploying a subgraph. It basically replaces the `network` and `address` fields in the `subgraph.yaml` file in place. The end result is that each time you do a deployment on a different network this file will be updated. So it's expected.